views/index.haml

```diff
@@ -1,13 +1,13 @@
-%p
-  SixNines is a 24x7 monitor of your website
-  availability. Just login, add one of your endpoints,
-  and in a few minutes you get a detailed report.
-  We ping all websites a few times per minute.
+SixNines is a 24x7 monitor of your website
```

Comment:
because we dont need top margin in this case. standard %p has top margin. just did not want to add more %p class without top margin.